doc/Architecture.md

```diff
@@ -312,22 +386,13 @@ Of course, we can write our own chain indexer, but it's probably better to build
 
 - [Interacting with ogmios](https://ogmios.dev/) which provides a convenient websockets to interact with the blockchain node, and we could update our database based on the information obtained by the websockets.
 
-As an ASCII diagram, we envision this to be
-
-```text
-                  
-                   Listens for events
- --------------                         ---------------          --------------
-| Cardano node | --------------------- | Chain indexer | ------ | SQL database |
- --------------                         ---------------          --------------
-            \                                 | 
-             \                                |
-               \                              | Query UTxOs
-      Submit txs \                            |
-                  --\                         | 
-                     \                   ------------ 
-                      \---------------- | Tx builder |        
-                                         ------------               
+
+```mermaid
+flowchart
+    Cardano[Cardano node]-- Listens for events ---CardanoIndexer[Chain indexer] ;
+    CardanoIndexer---SQL[(SQL database)];
+    CardanoIndexer-- Query UTxOs ---TXBuilder[Tx builder];
+    Cardano-- Submit Txs ---TXBuilder;
 ```
 
 where the lines denote communication, and the square boxes denote programs.
```

Comment:
````suggestion
    Cardano-- Submit Txs ---TXBuilder;
```

````
Let's get rid of the words below which explained what the diagram meant (since it's no longer lines and squares now)